test/docsystem.jl

```diff
@@ -0,0 +1,85 @@
+module DocSystemTests
+
+if VERSION >= v"0.5.0-dev+7720"
+    using Base.Test
+else
+    using BaseTestNext
+    const Test = BaseTestNext
+end
+
+import Documenter: Documenter, DocSystem
+
+const alias_of_getdocs = DocSystem.getdocs # NOTE: won't get docstrings if in a @testset
```

Comment:
I had to move this here, since the test on line 66 started failing (`d_4` had 0 docstrings).